app/controllers/courses/subjects_controller.rb

```diff
@@ -91,13 +91,14 @@ def build_course
     end
 
     def build_course_params
-      selected_master = params[:course][:master_subject_id]
+      selected_master = nil
```

Comment:
i think you can delete this line. it should be initialised to `nil` by default

Reply:
Thanks @asmega , done.